### ISSUE TITLE
Subscribe: Améliorer les messages d'erreurs

### ISSUE THREAD

**Issue by JMJHautier** [memOria-project/memOria#92]
quand l'utilisateur échoue à s'enregistrer sur le site, on leur affiche le message renvoyé par le serveur, par ex:
![grafik](https://user-images.githubusercontent.com/13255701/147969410-c7ce0b4e-79e5-43a8-b6f2-17f9a217ecac.png)
Ces messages ne sont pas toujours très claire. Ils pourraient être clarifiés, soit en front, soit en back. A voir.

**Comment by vmingam** [comment]
Je suis d'accord il faudrait rationaliser le traitement des erreurs et clarifier leur message en fonction des status
